examples/file_based_server/server_config.json5

```diff
@@ -0,0 +1,75 @@
+{
+  ServerConfig: {
```

Comment:
The entire file is a server config.

remove the outer object.
Also remove the outermost brackets.
Or parser does not need them.

Use comments in the first lines to explain this is JSON5 with a link.